zingolib/src/testutils/scenarios.rs

```diff
@@ -0,0 +1,965 @@
+//! In practice there are several common scenarios for which helpers are provided.
+//! These scenarios vary in the configuration of clients in use.  Most scenarios
+//! require some funds, the simplest way to access funds is to use a "faucet".
+//! A "faucet" is a client that receives mining rewards (because its spend capability
+//! generated the address registered as the `minetoaddress` in the zcash.conf that's
+//! used by the 'regetst mode' zcashs backing these tests.).
+//! HELPERS:
+//! If you just need a faucet, use the "faucet" helper.
+//! If you need a faucet, and a single recipient, use 'faucet_recipient`
+//! For less common client configurations use the client builder directly with
+//! custom_clients
+//! All scenarios have a default (i.e. faucet_default) which take minimal parameters and
```

Comment:
this was true until someone renamed `faucet_funded_recipient_default` to `orchard_funded_recipient`! non-blocking suggestion to put that name back so it matches this comment